docs/formatting/eslint.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Switch to ESLint
+description: How to change the default linter from Biome to ESLint.
+---
+
+import { Authors } from '/snippets/authors.mdx';
+
+<Authors data={[{
+  user: {
+    name: 'Hayden Bleasel',
+    id: 'haydenbleasel',
+  },
+  company: {
+    name: 'next-forge',
+    id: 'next-forge',
+  },
+}, {
+  user: {
+    name: 'Nicholas C. Zakas',
+    id: 'slicknet',
+  },
+  company: {
+    name: 'ESLint',
+    id: 'eslint',
+  },
+}]} />
+
+Here's how to switch from Biome to [ESLint](https://eslint.org). In this example, we'll also add the Next.js and React plugins, as well as the new ESLint Flat Config.
+
+## 1. Swap out the required dependencies
+
+First, uninstall the existing dependencies from the root `package.json` file...
+
+```sh Terminal
+pnpm remove -w @biomejs/biome ultracite
+```
+
+...and install the new ones:
+
+```sh Terminal
+pnpm add -w -D eslint eslint-plugin-next eslint-plugin-react eslint-plugin-react-hooks
+```
+
+## 2. Configure ESLint
+
+Delete the existing `biome.json` file in the root of the project, and create a new `eslint.config.js` file:
+
+```js eslint.config.js
+const react = require('eslint-plugin-react');
+const next = require('eslint-plugin-next');
+
+export default [
+  {
+    files: ['**/*.ts', '**/*.tsx'],
+    plugins: {
+      react: reactPlugin,
+      'react-hooks': hooksPlugin,
+      '@next/next': nextPlugin,
+    },
+    rules: {
+      ...reactPlugin.configs['jsx-runtime'].rules,
+      ...hooksPlugin.configs.recommended.rules,
+      ...nextPlugin.configs.recommended.rules,
+      ...nextPlugin.configs['core-web-vitals'].rules,
+      '@next/next/no-img-element': 'error',
+    },
+  },
+]
+```
+
+## 3. Install the ESLint VSCode extension
```

Comment:
This is generally installed if you selected "JavaScript" as a language to support when you first set up Visual Studio Code.